resources/measures/HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1113,28 +1113,26 @@ def self.process_ducts_for_unit(model, runner, ducts, building, unit, unit_index
 
     location_frac_conduction = location_frac_leakage
     ducts.num_returns = Airflow.get_num_returns(ducts.num_returns, num_stories)
-    supply_surface_area = Airflow.get_duct_supply_surface_area(ducts.supply_area_mult, unit_ffa, num_stories)
-    return_surface_area = Airflow.get_return_surface_area(ducts.return_area_mult, unit_ffa, num_stories, ducts.num_returns)
+    supply_surface_area = Airflow.get_duct_supply_surface_area(unit_ffa) * ducts.supply_area_mult
+    return_surface_area = Airflow.get_duct_return_surface_area(unit_ffa, ducts.num_returns) * ducts.return_area_mult
```

Comment:
Is this because Std 152 has a different method of calculating surface areas than BAHSP?

Reply:
I'm getting it from the OS-HPXML repo: https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/hvac.rb#L4017-L4031